src/toast/Toast.tsx

```diff
@@ -6,13 +6,20 @@ import { ToastTypes } from './types';
 interface Props {
   title?: string;
   description?: string;
+  // eslint-disable-next-line react/no-unused-prop-types
   id: string;
```

Comment:
Is `id` needed at all here, if it's not used?

Reply:
It was needed due the way Toast.tsx is used: by passing Toast object which has the id. This change is now outdated, because I changed the component to use Toast interface.